documentation/mkdocs.yml

```
@@ -109,27 +109,32 @@ markdown_extensions:
 # Navigation
 nav:
   - Home:                  index.md
-  - Architecture:          pages/Architecture.md
-  - NUOPC driver:          pages/NUOPC-driver.md
-  - Forcing data models:   pages/Forcing-data-models.md
-  - Configurations:
-    - pages/configurations/Overview.md
-    # At build time other configurations are added here by the mkdocs-include-configurations plugin
-  - Grids:                 pages/Grids.md
-  - Topography generation: pages/Topography-generation.md
-  - Initial condition generation: pages/Initial-conditions-generation.md
-  - Parameter choices: pages/Parameter-choices.md
-  - MOM6 generic tracers:  pages/MOM6-generic-tracers.md
-  - WW3:
-        - config: pages/MOM6-CICE6-WW3-quickstart.md
-        - workflow: pages/WW3_setup_and_workflow.md
-        - parameter choices: pages/ww3_parameter_choices.md 
-  - Git practices:         pages/Git-practices.md
+  - Contributing:          pages/Contributing.md
+  - Inputs:
+        - Forcing data models:   pages/Forcing-data-models.md
+        - Grids:                 pages/Grids.md
+        - Topography generation: pages/Topography-generation.md
+        - Initial condition generation: pages/Initial-conditions-generation.md
+        - ww3 workflow and input files: pages/WW3_setup_and_workflow.md
+
+  - Configuration choices:
+        - Configurations:
+          - pages/configurations/Overview.md
```

```suggestion
        - Configurations: pages/configurations/Overview.md
```
Can we remove the extra level here ?

---

I think this is best left as is for the moment. The "overview" gets effectively removed once the stubs are in, which will be re-added after this PR is merged.